pkg/fleet-manager/fleet_cluster.go

```diff
@@ -42,12 +43,16 @@ type ClusterKey struct {
 }
 
 func (f *FleetManager) buildFleetClusters(ctx context.Context, fleet *fleetapi.Fleet) (map[ClusterKey]*fleetCluster, error) {
```

Comment:
nit do we still need it

Reply:
`buildFleetClusters` method isn't called very often currently, so we can delete it and directly modify the places where it's invoked.